index.js

```diff
@@ -208,7 +208,9 @@ serveIndex.html = function _html(req, res, files, next, dir, showUp, icons, path
       // render html
       render(locals, function (err, body) {
         if (err) return next(err);
-        send(res, 'text/html', body)
+        if (res._header === null) {
```

Comment:
Please don't call an underscore prefixed property that is undocumented. Is there not a documented property to use?